pkg/storage/filesystem/btrfs.go

```diff
@@ -457,6 +484,21 @@ func (p *btrfsPool) Maintenance() error {
 	return nil
 }
 
+func (p *btrfsPool) Shutdown() error {
+	for _, device := range p.Devices() {
+		log.Info().Msgf("Shutting down disk %s ...", device.Path)
+		cmd := exec.Command("hdparm", "-y", device.Path)
```

Comment:
@DylanVerstraete @delandtj What happens if the nodes has some disk controller that cannot be controlled with hdparm ?

Reply:
Then this disk wont be shutdown, this needs to be investigated indeed